bouncer/views.py

```diff
@@ -90,14 +91,24 @@ def annotation(self):
         default_extension = settings["chrome_extension_id"]["default"]
         extension_id = settings["chrome_extension_id"].get(authority, default_extension)
 
+        # If a YouTube annotation has a media time associated, this means it
+        # was made using Via's transcript annotation tool.
+        #
+        # This means we force the use of Via, even if the extension is
+        # installed.
+        always_use_via = False
+        if document_uri.startswith("https://www.youtube.com") and has_media_time:
+            always_use_via = True
```

Comment:
We will have to update this if Via's video player gains support for other video sites in future. At that point it would be better to put a more specific marker in the annotation itself indicating which tool was used.